scss/_off.scss

```diff
@@ -386,3 +386,11 @@ a[aria-controls="idOfLeftMenu"] {
   }
 
 }
+
+@media only screen and (max-width: 28.125em) {
```

Comment:
Thank you for the PR!

`max-width: 28.125em`

Where did you take this value from? Are we sure that it's good for every mobile device?

Otherwise all good!

Reply:
Here is what I found on this subject: [https://css-tricks.com/em-based-media-queries-are-based-on/](https://css-tricks.com/em-based-media-queries-are-based-on/). I think using `450px` here would be better: it is more readable and as I understood, browser settings may affect media query with `em`

Reply:
Examples: 

Font set to "small" in chromium settings: 

![screenshot](https://screenshots.gennady.pp.ua/20200514_b5f67a6c0.png)

Font set to "medium" (recommended and default value):

![screenshot](https://screenshots.gennady.pp.ua/20200514_a585f067b.png)

Reply:
Maybe the breakpoint variables from Foundation could be used?

Reply:
> 
> 
> Maybe the breakpoint variables from Foundation could be used?

We're still on Foundation 5, so we should do as [the documentation](https://get.foundation/sites/docs/media-queries.html#upgrading-from-foundation-5) says.

Eg.
```sass
@media #{$medium-only} {
}
```

Reply:
Hi! I just assumed that em was used based on the other media queries in the file. I converted 450px to em, which was 28.125 em. And I'm not sure if all devices are under 450px width (majority of them are). But I noticed that at that width, the content starts to shift to one side.

<img width="489" alt="above_450" src="https://user-images.githubusercontent.com/21044058/81954411-d0fcf480-95d6-11ea-83dd-ea8f680a3428.png">
<img width="497" alt="below_450" src="https://user-images.githubusercontent.com/21044058/81954417-d35f4e80-95d6-11ea-9e51-2b75dc70a2bf.png">

Reply:
Let's use $small-only